common/app/navigation/NavLinks.scala

```diff
@@ -594,7 +594,7 @@ object NavLinks {
   )
 
   val ukBrandExtensions = List(
-    jobs.copy(url = jobs.url + "?INTCMP=jobs_uk_web_newheader_dropdown"),
+    jobs,
```

Comment:
[`jobs` have the link without `INTCMP`](https://github.com/guardian/frontend/blob/main/common/app/navigation/NavLinks.scala#L225)